app/gather/templates/base.html

```diff
@@ -26,10 +27,11 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <meta name="robots" content="NONE,NOARCHIVE" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
-    <link href="https://fonts.googleapis.com/css?family=Asap:400,400i|Quicksand:400,500,700" rel="stylesheet">
+    <meta name="description" content="{% trans 'Survey collection and analytics' %}">
```

Comment:
'Effortless data collection and curation'

Reply:
In `README` too?